src/ucs/config/parser.c

```diff
@@ -693,7 +693,7 @@ int ucs_config_sscanf_range_spec(const char *buf, void *dest, const void *arg)
     char *p, *str;
     int ret = 1;
 
-    str = strdup(buf);
+    str = ucs_strdup(buf, "config_sscanf_range_spec str");
```

Comment:
maybe add ucs_strdupa() macro to use ucs_alloca + memcpy() ?

Reply:
alloca on processing of external data may cause stack overflow. do you think it is reasonable to use here?

Reply:
ok
so can you pls shorter allocation names for ucs_strdup?
for example, config_range_spec_str

Reply:
renamed